osu.Game/Online/Spectator/SpectatorStreamingClient.cs

```diff
@@ -82,114 +75,50 @@ public class SpectatorStreamingClient : Component, ISpectatorClient
         /// </summary>
         public event Action<int, SpectatorState> OnUserFinishedPlaying;
 
-        private readonly string endpoint;
-
         public SpectatorStreamingClient(EndpointConfiguration endpoints)
         {
-            endpoint = endpoints.SpectatorEndpointUrl;
+            InternalChild = connector = new HubClientConnector("Spectator client", endpoints.SpectatorEndpointUrl)
```

Comment:
```suggestion
            InternalChild = connector = new HubClientConnector("spectator", endpoints.SpectatorEndpointUrl)
```

or `nameof(SpectatorStreamingClient)`.